pom.xml

```diff
@@ -10,7 +10,7 @@
     </description>
     <url>https://github.com/evis/scalafix-maven-plugin</url>
     <groupId>io.github.evis</groupId>
-    <artifactId>scalafix-maven-plugin_${scala.major.version}</artifactId>
+    <artifactId>scalafix-maven-plugin_${scala.binary.version}</artifactId>
```

Comment:
why is this change needed?

Reply:
I tried to follow the conventions from [here](https://github.com/kelnos/scala-cross-maven-plugin), but I reverted them.